starwars/src/components/Data.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './StarWars.css';
+
+function Data(props) {
+  return (
+    <div>
+      {props.getCharacters}
```

Comment:
Isn't `getCharacters` supposed to be a function? We can't render a function definition (you'd have to invoke it)!